elementary/monitor/dbt_project/macros/artifacts_queries/dbt_models_query.sql

```diff
@@ -0,0 +1,9 @@
+{% macro dbt_models_query() %}
```

Comment:
My intuition is that in this case as it's simply filtering the package maybe it's better to just have the package != elementary in all 4 places?